### PR TITLE
[DevServer] Authentication Issues

### PR DESCRIPTION
**Pull request by ragg09-sunasterisk**
## Issue Link
[LMS-703 [DevServer] Authentication issues](https://framgiaph.backlog.com/view/LMS-703)

## Defintion of Done
- can login using  allowed user

## Steps to reproduce
- visit https://sph-lms.vercel.app
- login
- in sign in prompt, use this temporary credential username = `admin` & passowrd = `password`

## Affected Components / Functionalities / Page
- auth_view.py

## Test Cases
_will update soon..._

## Notes
- this is not a solution but a bypass, the actual solution is for improvement. We will be needing the changes in this PR to login in backend so that we can proceed to UAT.
Basically, the changes is for UAT purposes only, and the fix for the issue will be included for improvements.

## Screenshots
n/a

### CODE REVIEW

**Review by nicoleamber — APPROVED** [review]

**Review by Melanthonyando — APPROVED** [review]
### Approved 👍 

Iteration: 0
Comment: 0 
Testing Time : 30 mins.
